SDK/AppCenterAuth/Microsoft.AppCenter.Auth.Android/Auth.cs

```diff
@@ -22,7 +35,8 @@ private static Task<UserInformation> PlatformSignInAsync()
                 var signInResult = (SignInResult)future.Get();
                 if (signInResult.Exception != null)
                 {
-                    throw signInResult.Exception;
+                    // Keep the stacktrace clean.
```

Comment:
It would be nice to add a comment to note the behavior difference between native android and xamarin.android that one throws and one doesn't.

Reply:
The behavior is the same - both throwing the exception. But iOS doesn't have C# `Exception` object to (re)throw, so we don't need to keep the previous stacktrace there.

Reply:
@MatkovIvan Sorry I wan't clear here. I mean native android sign-in API which doesn't throw but holds the exception at `signInResult.Exception` that is different from xamarin android which throws exception on sign-in API.

Reply:
It's documented in the shared code that it throws.
We have shared code for all platforms that calls the same method name with Platform prefix.

Reply:
Sounds good.